src/main/java/org/kainos/ea/utility/DatabaseConnector.java

```diff
@@ -21,24 +22,17 @@ public Connection getConnection() throws SQLException, DatabaseConnectionExcepti
         }
 
         try {
-            dbCredentials = new DbCredentials();
-            user = dbCredentials.getDbUser();
-            password = dbCredentials.getDbPassword();
-            host = dbCredentials.getDbHost();
-            database = dbCredentials.getDbName();
+            user            = System.getenv("DB_USERNAME");
+            password        = System.getenv("DB_PASSWORD");
+            host            = System.getenv("DB_HOST");
+            database        = System.getenv("DB_NAME");
```

Comment:
should this have been changed?

Reply:
Should this be using the DbCredentials object or is that causing some issues?

Reply:
I'm not sure why this has been changed

Reply:
think it was just a merging problem and i accidentally removed the wrong thing ill fix this now